docs/reference/commands/run.rst

```diff
@@ -137,6 +137,43 @@ contains the most recent test code. To prevent this update and build, use the
 Prevent the automated update and build of app code that is performed when
 specifying by the ``--test`` option.
 
+``--debug <debugger>``
+----------------------
+
+Run the app in debug mode in the bundled app environment and establish an
+debugger connection via a socket.
+
+Currently the following debuggers are supported:
+
+- ``pdb``: This is used for debugging via console (see :doc:`Debug via Console </how-to/debugging/console>`)
+- ``debugpy``: This is used for debugging via VSCode (see :doc:`Debug via VSCode </how-to/debugging/vscode>`)
+
+For ``debugpy`` there is also a mapping of the source code from your bundled
+app to your local copy of the apps source code in the ``build`` folder. This
+is useful for devices like iOS and Android, where the running source code is
+not available on the host system.
```

Comment:
Why are we telling the user this? Does it make a difference to anything they do?